charts/pulsar/templates/proxy-configmap.yaml

```diff
@@ -27,6 +27,15 @@ metadata:
     {{- include "pulsar.standardLabels" . | nindent 4 }}
     component: {{ .Values.proxy.component }}
 data:
+  # Metadata settings
+  zookeeperServers: "{{ template "pulsar.zookeeper.connect" . }}{{ .Values.metadataPrefix }}"
+  {{- if .Values.pulsar_metadata.configurationStore }}
+  configurationStoreServers: "{{ template "pulsar.configurationStore.connect" . }}{{ .Values.pulsar_metadata.configurationStoreMetadataPrefix }}"
+  {{- end }}
+  {{- if not .Values.pulsar_metadata.configurationStore }}
```

Comment:
Nit:
```suggestion
  {{- else }}
```

Reply:
Thanks, that makes sense. I had copy-pasted the original logic from https://github.com/apache/pulsar-helm-chart/blob/dc97bd4ac6ae59610f685341dedd716ed10a8165/charts/pulsar/templates/broker-configmap.yaml#L30-L37